config.go

```diff
@@ -18,11 +18,6 @@ type Config struct {
 	// a leader before we attempt an election.
 	ElectionTimeout time.Duration
 
-	// CommitTimeout controls the time without an Apply() operation
```

Comment:
The timing is a lot more aggressive than a heartbeat and given how Consul works with allowing stale reads this is probably pretty important for good performance there. Accelerating heartbeats seems equivalent. Still working through the diff so I'm not sure if that would lead to an unclean coupling of things.